src/ca65/instr.c

```diff
@@ -783,7 +788,7 @@ static const InsTable* InsTabs[CPU_COUNT] = {
     (const InsTable*) &InsTab65816,
     (const InsTable*) &InsTabSweet16,
     (const InsTable*) &InsTabHuC6280,
-    0,                                  /* Mitsubishi 740 */
+    0,
```

Comment:
Responding to the original comment, I think the comment should stay there, unless we remove the incomplete support for the Misubishi/Renesas chip altogether.